src/tasks/registerTaskProviders.ts

```diff
@@ -0,0 +1,37 @@
+import { ExtensionContext, tasks } from 'vscode';
+import { DockerBuildTaskProvider } from './DockerBuildTaskProvider';
+import { DockerComposeTaskProvider } from './DockerComposeTaskProvider';
+import { DockerRunTaskProvider } from './DockerRunTaskProvider';
+import { netCoreTaskHelper } from './netcore/NetCoreTaskHelper';
+import { nodeTaskHelper } from './node/NodeTaskHelper';
+import { pythonTaskHelper } from './python/PythonTaskHelper';
+
+
+export function registerTaskProviders(ctx: ExtensionContext): void {
```

Comment:
Consider moving this file into the "utils" folder with other helper functions.

Reply:
:satisfied: